### DISCUSSION TITLE
Custom Windows power modes, including Charging/On Battery

### DISCUSSION THREAD

**Discussion by justing6** [seerge/g-helper#608] [Ideas]
Hello,

Right now, "Auto adjust Windows Power Modes" sets "Best power efficiency" for Silent, "Balanced" for Balanced, and "Best Performance" for Turbo. Given how this effects how intel CPUs turbo, I think it would be great to be able to customize these and have different assignments while charging vs on battery. For example, I prefer how Armory Crate maps these by default:

On Battery:
Silent: Best power efficiency
Balanced: Best Performance
Turbo: (not available on AC on battery)

Charging:
Silent: Balanced
Balanced: Best Performance
Turbo: Best Performance

**Comment by seerge** [comment]
Hello, i don't think that you are right in a first place. Armoury Crate doesn't adjust power modes at all. As it uses own (custom) power plans instead.

And each mode has own power plan, that doesn't change depending on plugged / battery state :)

**Comment by justing6** [comment]
Hello, I don't have a good way of sending video here, so I dm'd you a video of the original Armory Crate behavior I'm suggesting in Discord.

**Reply by seerge** [reply]
I have armoury :) and know what it does
